src/protocols/secure_channel/CASESession.cpp

```diff
@@ -981,39 +1114,15 @@ CHIP_ERROR CASESession::HandleSigmaR3(System::PacketBufferHandle && msg)
     mDelegate->OnSessionEstablished();
```

Comment:
Just above: same comment as for Sigma2 handling: remove usage of "pairing" wording.